src/modules/commander/Commander.cpp

```diff
@@ -2493,6 +2488,11 @@ Commander::run()
 
 		_status_changed = false;
 
+		/* store last position lock state */
```

Comment:
Those are just moved for structure right? I don't see any functional difference.

Reply:
Yes I've just started collecting any previous state flags at the end of the loop to prevent surprises.